README.md

```diff
@@ -8,6 +8,16 @@
 This plugin deals with the authentication and fetching of the JSON schema from a live Jenkins instance.
 It provides the base for validation of yaml files written to configure a Jenkins instance.
 
+## How to use
+
+1) Download and install the plugin from the VSCode Marketplace.
+2) Inside the Settings tab of VSCode find JCasc which will display two input blocks:
+    a) URL fileld for the Jenkins instance that you want to to extract the YAML Schema from.
```

Comment:
formatting is off:
![image](https://user-images.githubusercontent.com/21194782/69434363-5f35a380-0d35-11ea-96ab-c623c7fc7105.png)

Reply:
Yeah forgot the preview completely :(